test/routing/Page.test.tsx

```diff
@@ -63,7 +63,7 @@ describe.skip('Routing - Page', () => {
       </MemoryRouter>
     );
     act(() => {
-      const goLink = container.querySelector('button.next');
+      const goLink = container.querySelector('va-button[continue][submit]');
```

Comment:
There's only one `<va-button>` rendered here, I'm just using the attribute selectors to be a bit more specific so we can tell that we're working with this one:

https://github.com/department-of-veterans-affairs/va-forms-system-core/blob/3fe200c11b3430367b154693fd7e635fa17033d6/src/routing/Page.tsx#L62-L64

Rather than this one:

https://github.com/department-of-veterans-affairs/va-forms-system-core/blob/3fe200c11b3430367b154693fd7e635fa17033d6/src/routing/Page.tsx#L47-L57